nrf_modem/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if(CONFIG_NRF_MODEM_LINK_WITH_BINARY)
+
+nrfxlib_calculate_lib_path(lib_path)
+
+set(NRF_MODEM_PATH ${CMAKE_CURRENT_SOURCE_DIR}/${lib_path})
+
+if(NOT EXISTS ${NRF_MODEM_PATH})
+  message(WARNING "This combination of SoC and floating point ABI is not supported by libmodem."
+                  "(${NRF_MODEM_PATH} doesn't exist.)")
```

Comment:
Why is this only a warning ?

This will definitely during link time, so why not fail fast here and save some headache for the user.